docs/GITCOINBOT.md

```diff
@@ -0,0 +1,39 @@
+# Gitcoinbot is a Github App for Gitcoin.co
+
+
+<a href="https://github.com/apps/gitcoinbot">
+    <img src="https://github.com/gitcoinco/web/raw/master/app/gitcoinbot/install.jpg">
+</a>
+
+## Usage instructions
+
+[Gitcoinbot](https://github.com/Gitcoinbot) is a bot that allows you to interact with gitcoin via the github comments, like follows:
+
+### Help Command
+![Gitcoinbot help](https://i.postimg.cc/tC2XHSFf/ezgif-1-defad716e014.gif)
```

Comment:
Can you include these gifs in the repo itself? Thanks!

Reply:
> Can you include these gifs in the repo itself? Thanks!

Sure thing my buddy. I was thinking something like this. We should not trust other hosting servers for our files/images

Reply:
@molecula451 Theres another one 2 lines down 🙇  thanks!

Reply:
@danlipert Uploaded and updated my friend. I have pointed out directly to the gitcoin/web repo. So at it soons it is merge it should detect the images that are in the original repo. Take a look